Tracking/src/ClicEfficiencyCalculator.cc

```diff
@@ -611,7 +612,7 @@ void ClicEfficiencyCalculator::processEvent( LCEvent* evt ) {
     if(particleHits.count(particle) == 0) continue;
     
     // Cut on stable particles
-    if(particle->getGeneratorStatus() != 1) continue;
+    //if(particle->getGeneratorStatus() != 1) continue;
```

Comment:
Should we remove these two lines? I suppose this is needed in the 'NHits, NHitsVXD' options (while in 'ILDLike' is already included)